### PR TITLE
Fix bytecode backtrace generation when large integers are present.

### PR DESCRIPTION
**Pull request by stedolan**
#9268 again, since @jhjourdan noticed I only fixed half of the problem there.

### CODE REVIEW

**Comment by gasche** [comment]
@jhjourdan Stephen wins by seven minutes. Would you review his version of your patch?

File: runtime/backtrace_byt.c [hunk 1]
```diff
@@ -260,7 +260,10 @@ void caml_stash_backtrace(value exn, value * sp, int reraise)
 code_t caml_next_frame_pointer(value ** sp, value ** trsp)
 {
   while (*sp < Caml_state->stack_high) {
-    code_t *p = (code_t*) (*sp)++;
+    value spv = (*sp)++;
```
Comment (jhjourdan) [inline runtime/backtrace_byt.c]: `spv` is of type `value*`, right?
```suggestion
    value *spv = (*sp)++;
```

File: runtime/backtrace_byt.c [hunk 2]
```diff
@@ -260,7 +260,10 @@ void caml_stash_backtrace(value exn, value * sp, int reraise)
...
+    code_t *p;
+    if (Is_long(spv)) continue;
```
Comment (jhjourdan) [inline runtime/backtrace_byt.c]: So it should be dereferenced here.
```suggestion
    if (Is_long(*spv)) continue;
```

**Review by jhjourdan — CHANGES_REQUESTED** [review]

**Comment by stedolan** [comment]
> @jhjourdan Stephen wins by seven minutes.

The important thing, of course, is how soon the code is pushed. Building and/or working are optional!

**Review by jhjourdan — APPROVED** [review]
I think the code is now good to merge.

**Review by gasche — APPROVED** [review]
Approved on @jhjourdan's behalf. I will now merge and cherry-pick in 4.10 ( 368ee3cd74a80d5609a8bb7acae512001c7c5736 ).

**Pull request merged**